merge-by/action.yml

```diff
@@ -0,0 +1,37 @@
+name: "Merge-by"
+description: "Tracks the status of ongoing pull requests and provides expected dates for merging"
+
+inputs:
+  operation:
+    description: 'The operation to run as part of this action (build-table, bump-dates)'
+    required: true
+    default: 'build-table'
+
+runs:
+  using: "composite"
+  steps:
+    - uses: actions/checkout@v3
+    - uses: pnpm/action-setup@v4
+      with:
+        version: 8
+        run_install: false
+    - uses: actions/setup-node@v4
+      with:
+        node-version: 20
+        cache: 'pnpm'
+    - run: pnpm install
+      shell: "sh"
```

Comment:
I ended up removing the node cache because it doesn't play well with the context in which the action is run.
Instead, I install the `dayjs` NPM package globally and then run the scripts.